.github/actions/run-synthetics-stack-e2e/action.yml

```diff
@@ -0,0 +1,137 @@
+      uses: elastic/apm-pipeline-library/.github/actions/is-member-elastic-org@current
+      with:
+        username: ${{ inputs.user }}
+        token: ${{ env.GITHUB_TOKEN }}
+
+    - name: Notify with a reaction
+      continue-on-error: true
+      uses: elastic/apm-pipeline-library/.github/actions/comment-reaction@current
+      with:
+        repository: ${{ inputs.repository }}
+        commentId: ${{ inputs.comment-id }}
+        emoji: ${{ steps.is_elastic_member.outputs.result && '+1' || '-1' }}
+        token: ${{ env.GITHUB_TOKEN }}
+
+    - name: Exit if not an Elastician comment
+      if: steps.is_elastic_member.outputs.result == 'false'
+      run: exit 1;
+      shell: bash
+
+    - uses: elastic/apm-pipeline-library/.github/actions/docker-login@current
+      with:
+        registry: docker.elastic.co
+        secret: secret/observability-team/ci/docker-registry/prod
+        url: ${{ inputs.vault-url }}
+        roleId: ${{ inputs.vault-role-id }}
+        secretId: ${{ inputs.vault-secret-id }}
+
+    - uses: elastic/apm-pipeline-library/.github/actions/kibana-docker-image@current
+      id: kibana-docker-image
+      with:
+        github-repository: ${{ inputs.repository }}
+        git-ref: refs/pull/${{ inputs.pr-number }}/head
+
+    - name: Create cluster
+      id: create-cluster
+      run: >-
+        ${{ github.action_path }}/create-cluster.sh
+        ${{ steps.kibana-docker-image.outputs.kibana-docker-image }}
+        ${{ steps.kibana-docker-image.outputs.kibana-stack-version }}
+      shell: bash
+
+    # - uses: elastic/apm-pipeline-library/.github/actions/buildkite@current # TODO: Use this when PR is merged
+    - uses: reakaleek/apm-pipeline-library/.github/actions/buildkite@feature/buildkite-branch-input
+      id: buildkite
+      with:
+        vaultUrl: ${{ inputs.vault-url }}
+        vaultRoleId: ${{ inputs.vault-role-id }}
+        vaultSecretId: ${{ inputs.vault-secret-id }}
+        pipeline: ${{ inputs.buildkite-pipeline-slug }}
+        triggerMessage: Triggered by PR ${{ inputs.repository }}#${{ inputs.pr-number }}
+        pipelineBranch: use-kibana-url
+        waitFor: true
+        buildEnvVars: |
+          OBLT_CLUSTER_STATE_SECRET_PATH=secret/observability-team/ci/test-clusters/${{ steps.create-cluster.outputs.cluster-name }}/cluster-state
+          KIBANA_PR_URL=${{ github.server_url }}/${{ inputs.repository }}/pull/${{ inputs.pr-number }}
+
+    - if: always() && steps.buildkite.outputs.build
+      name: Add notice message
+      run: |
+        echo "::notice title=${TITLE}::${BUILDKITE_BUILD_URL}"
+      shell: bash
+      env:
+        BUILDKITE_BUILD_URL: ${{ steps.buildkite.outputs.build }}
+        TITLE: Synthetics Stack E2E Buildkite Build
+
+    - if: always() && steps.create-cluster.outcome == 'success'
+      name: Destroy cluster
+      run: >-
+        ${{ github.action_path }}/destroy-cluster.sh
+        ${{ steps.create-cluster.outputs.cluster-name }}
```

Comment:
no particular reason, just that there is a create-cluster.sh script and it felt naturally to create a destroy-cluster.sh script.

will remove it.. since it's a single command script.

Reply:
https://github.com/elastic/apm-pipeline-library/pull/2367/commits/2db7aee95e385d8d6bc5845ecb3e9feea2cdc1a2